### ISSUE TITLE
Shadows are turned on, but shadows are not displayed on the map tiles.

### ISSUE THREAD

**Issue by datouzhan** [heremaps/harp.gl#2123]
**Is your feature request related to a problem? Please describe.**
I have configured the theme json
![image](https://user-images.githubusercontent.com/20176393/109095210-46951400-7756-11eb-9c9b-e14683826109.png)
And set the light.Shadows are turned on, but shadows are not displayed on the map tiles.
![image](https://user-images.githubusercontent.com/20176393/109095745-4b0dfc80-7757-11eb-8aa5-af498adc8b09.png)

**Describe the solution you'd like**
![image](https://user-images.githubusercontent.com/20176393/109095377-8eb43680-7756-11eb-8f6e-1099f5e8c1e7.png)
shadows are displayed on the map tiles.

**Comment by FraukeF** [comment]
Hi @datouzhan,

did you enable the shadows in MapView too, something like:
`  map = new MapView({
        canvas,
        theme,
        enableShadows: true
    });`

This is needed to enable the shadow on the map materials too.

Let me know if this helps.

Frauke

**Comment by datouzhan** [comment]
Yes,I've already set it enable.But it didn't help.The map loaded is WebTileDataSource. if it has anything to do with this.

**Comment by datouzhan** [comment]
Yes,I've already set it enable.But it didn't help.The map loaded is WebTileDataSource. if it has anything to do with this.

**Comment by FraukeF** [comment]
ah sorry, I should have seen that from the screenshot probably, ... it will only work with a VectorTileDataSource below

**Comment by datouzhan** [comment]
This is really sad news. if I want the map tiles to also show shadows, do I need to expand it?

**Comment by FraukeF** [comment]
Hi, I just checked, whats actually missing to make it work... I created some draft PR so you can try it out https://github.com/heremaps/harp.gl/pull/2125/commits .. I ll try to polish it later and push it to master

**Comment by datouzhan** [comment]
I'm glad to hear that.  @FraukeF , When will the next version be built?

**Comment by nzjony** [comment]
@datouzhan , @FraukeF  has a merged PR: https://github.com/heremaps/harp.gl/pull/2125

We will do a release in the next week, so stay tuned.
